lib/extensions/ar_migration.rb

```diff
@@ -0,0 +1,46 @@
+module MigrationSyncHelper
+  def self.migrations_column_present?
+    ActiveRecord::Base.connection.columns("miq_regions").map(&:name).include?("migrations_ran")
```

Comment:
Would `detect` be better? `ActiveRecord::Base.connection.columns("miq_regions").detect { |column| column.name == "migrations_ran" }`

Reply:
Also, should we cache this?  Once the column is found, we shouldn't have to look for it anymore.

Reply:
I don't really have a preference so I can change it to the detect version as it looks slightly more performant.

I don't think we should cache because I think it would be shared across multiple calls to `migrate` which would defeat the purpose.